ab_tool/views/intervention_point_pages.py

```diff
@@ -158,7 +158,7 @@ def edit_intervention_point_common(request, intervention_point_id):
     notes = post_param(request, "notes")
     intervention_point.update(name=name, notes=notes)
     # Validates URLs using backend rules before any InterventionPointUrl object creation
-    intervention_pointurls = [(k,validate_format_url(v)) for (k,v) in request.POST.iteritems() if STAGE_URL_TAG in k and v]
+    intervention_pointurls = [(k,validate_format_url(v.strip())) for (k,v) in request.POST.iteritems() if STAGE_URL_TAG in k and v]
```

Comment:
.strip() should be moved into the function `validate_format_url` because `validate_format_url(v)` is also elsewhere, such as line 86. By moving into the function, you can be assured that future calls will include whitespace stripping.